src/types.ts

```diff
@@ -69,3 +66,38 @@ interface AragonEnvironment {
   registry: string
   wsRPC: string
 }
+
+/**
+ * App Installer types
+ */
+
+export type NetworkType =
+  | 'homestead'
+  | 'rinkeby'
+  | 'ropsten'
+  | 'kovan'
+  | 'goerli'
+
+export type AppInstaller = (
+  name: string,
+  appOptions?: AppOptions
+) => Promise<AppInstalled>
+
+export interface AppInstalled {
+  initialize: (_initializeArgs: any[]) => Promise<void>
+  createPermission: (roleName: string, entity?: string) => Promise<void>
+  address: string
+}
+
+export interface AppOptions {
```

Comment:
A bit of a nitpick, but I think it might be easier to read if the argument types were placed ahead of the return types, as they're declared in that order in the function definition.